pkg/webhooks/utils/policy_context_builder.go

```diff
@@ -115,13 +57,12 @@ func (b *policyContextBuilder) Build(request *admissionv1.AdmissionRequest, poli
 	userRequestInfo := kyvernov1beta1.RequestInfo{
 		AdmissionUserInfo: *request.UserInfo.DeepCopy(),
 	}
-	if containsRBACInfo(policies...) {
-		var err error
-		userRequestInfo.Roles, userRequestInfo.ClusterRoles, err = userinfo.GetRoleRef(b.rbLister, b.crbLister, request, b.configuration)
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to fetch RBAC information for request")
-		}
+	var err error
+	userRequestInfo.Roles, userRequestInfo.ClusterRoles, err = userinfo.GetRoleRef(b.rbLister, b.crbLister, request, b.configuration)
```

Comment:
can you assign err in the if clause ?

Reply:
done

Reply:
On the left side, `userRequestInfo` is already initialised. so it will not allow it.

Reply:
Ok, let's do that later.